analyzers/src/SonarAnalyzer.CSharp/Rules/AvoidExcessiveClassCoupling.cs

```diff
@@ -112,7 +112,8 @@ public sealed class AvoidExcessiveClassCoupling : ParameterLoadingDiagnosticAnal
                 SyntaxKind.ClassDeclaration,
                 SyntaxKind.StructDeclaration,
                 SyntaxKind.InterfaceDeclaration,
-                SyntaxKindEx.RecordClassDeclaration);
+                SyntaxKindEx.RecordClassDeclaration,
+                SyntaxKindEx.RecordStructDeclaration);
```

Comment:
_Line 89 above:_ Can we use the new extension there to explain why the condition is there?